views/view_package.dt

```diff
@@ -10,6 +10,28 @@ block title
 	- import userman.controller;
 	- auto title = "Package " ~ packageName ~ " version " ~ versionInfo["version"].get!string;
 
+	- auto normalizedPackageName = packageName.splitter(':').front;
```

Comment:
`findSplit(':')[0]`

Reply:
What's the advantage of findSplit?

Reply:
@wilzbach oh actually I just found out that if a `packageName` were to be empty, this method would crash because split and splitter return an empty array for an empty string...

100 comments on this pr :tada:

Reply:
Yeah, but that's not going to happen, is it?
Anyhow, if you are experiencing this problem somewhere else, there you might like `orElse`

https://github.com/dlang/phobos/pull/5154

I'm also a big fan or `safeDeref`, but I'm not sure whether the version on the NG supports ranges out of the box.